examples/run_fine_tuning.py

```diff
@@ -247,7 +253,12 @@ def get_model_preds_and_references(model, validation_stream):
     for datum in tqdm(validation_stream):
         # Local .run() currently prepends the input text to the generated string;
         # Ensure that we're just splitting the first predicted token & beyond.
-        raw_model_text = model.run(datum.input).text
+        # TEMP HACK to avoid too big input to TGIS problem:
+        try:
+            raw_model_text = model.run(datum.input).generated_text
+        except Exception as ex:
+            print(ex)
```

Comment:
oof. Any way to make this more narrow, or is it like a general exception? Or, should we at least catch separately for in vs not in TGIS?